From 4306c9173eab53cb1ea33382bee265f29b398c26 Mon Sep 17 00:00:00 2001
From: Mousumi Jana <mousumix.jana@intel.com>
Date: Thu, 28 Apr 2016 02:27:21 +0530
Subject: [PATCH 1951/2508] ASoC: Intel: Boards: Add probe DAI Link in IVI HU
 Machine driver.

commit aaeb32fcbe3f74997a5f7484ec5c8e9d48c11612 from
https://github.com/01org/linux-apollolake-i

Introduce the DAI Link for probe in HU machine driver.

Change-Id: Ie0330c219be22f7f537c26497a2b531d282aacb5
Signed-off-by: Mousumi Jana <mousumix.jana@intel.com>
---
 sound/soc/intel/boards/bxtp_ivi_hu_rt298.c |   22 ++++++++--------------
 1 files changed, 8 insertions(+), 14 deletions(-)

diff --git a/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c b/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c
index ac63150..ab5a542 100644
--- a/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c
+++ b/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c
@@ -78,8 +78,6 @@ static const struct snd_soc_dapm_widget broxton_widgets[] = {
 	SND_SOC_DAPM_MIC("Mic Jack", NULL),
 	SND_SOC_DAPM_MIC("DMIC2", NULL),
 	SND_SOC_DAPM_MIC("SoC DMIC", NULL),
-        SND_SOC_DAPM_SIGGEN("probe source"),
-        SND_SOC_DAPM_SINK("probe sink"),
 };
 
 static const struct snd_soc_dapm_route broxton_rt298_map[] = {
@@ -91,9 +89,6 @@ static const struct snd_soc_dapm_route broxton_rt298_map[] = {
 
 	{ "bt_ssp0_in", NULL, "ssp0 Rx" },
 	{ "ssp0 Rx", NULL, "Dummy Capture" },
-        { "extractor cpr 5", NULL, "probe source"},
-        { "probe sink", NULL, "injector cpr 4"},
-
 };
 
 static int bxtp_ssp0_gpio_init(struct snd_soc_pcm_runtime *rtd)
@@ -183,29 +178,28 @@ static struct snd_soc_dai_link broxton_rt298_dais[] = {
 		.capture_only = true,
 		.ignore_suspend = 1,
 	},
-        {
-                .name = "Bxt Probe playback",
-                .stream_name = "Probe Playback",
-                .cpu_dai_name = "Probe0 Pin",
+	{
+		.name = "Bxt Compress Probe playback",
+		.stream_name = "Probe Playback",
+		.cpu_dai_name = "Compress Probe0 Pin",
                 .codec_name = "snd-soc-dummy",
                 .codec_dai_name = "snd-soc-dummy-dai",
                 .platform_name = "0000:00:0e.0",
                 .init = NULL,
-                .ignore_suspend = 1,
                 .nonatomic = 1,
         },
         {
-                .name = "Bxt Probe capture",
-                .stream_name = "Probe Capture",
-                .cpu_dai_name = "Probe1 Pin",
+		.name = "Bxt Compress Probe capture",
+		.stream_name = "Probe Capture",
+		.cpu_dai_name = "Compress Probe1 Pin",
                 .codec_name = "snd-soc-dummy",
                 .codec_dai_name = "snd-soc-dummy-dai",
                 .platform_name = "0000:00:0e.0",
                 .init = NULL,
-                .ignore_suspend = 1,
                 .nonatomic = 1,
         },
 
+
 	/* Back End DAI links */
 	{
 		/* SSP0 - Codec - for HDMI MCH */
-- 
1.7.5.4

